riscv_isac/coverage.py

```diff
@@ -646,9 +697,10 @@ def compute_per_line(instr, cgf, xlen, addr_pairs,  sig_addrs):
             if store_address >= start and store_address <= end:
                 sig_update = True
                 break        
-
+    #This flag is a temproray aid to the data propagation (as the current implementation needs more inprovement which is being worked on),
+    #This can be removed including the flag swFlag upon fixing the issue #26
     for cov_labels,value in cgf.items():
-        if instr.instr_name in ['sw','sd'] and str(value['opcode']).split("'")[1] in ['fclass.s','fclass.d','fcvt.w.d','fcvt.wu.d','feq.d','fle.d','flt.d','fcvt.l.s','fcvt.lu.s','fcvt.wu.s','fcvt.w.s','fle.s','feq.s','flt.s','fmv.x.w'] and stats.swFlag:
+        if instr.instr_name in ['sw','sd'] and str(value['opcode']).split("'")[1] in ['fclass.s','fclass.d','fcvt.w.d','fcvt.wu.d','feq.d','fle.d','flt.d','fcvt.l.s','fcvt.lu.s','fcvt.wu.s','fcvt.w.s','fle.s','feq.s','flt.s','fmv.x.w','fcvt.l.d','fcvt.lu.d','fmv.x.d'] and stats.swFlag:
```

Comment:
This is an unwanted change just to avoid a few error lines in the log. Detecting a store to a signature region without a signature update is not a critical error. The coverage computation still proceeds without failure.